base/common/DateLUTImpl.h

```diff
@@ -474,38 +493,23 @@ class DateLUTImpl
         /// also make the special timezones with no whole hour offset such as 'Australia/Lord_Howe' been taken into account.
 
         LUTIndex index = findIndex(t);
-        UInt32 time = t - lut[index].date;
+        Time time = t - lut[index].date;
 
         if (time >= lut[index].time_at_offset_change())
             time += lut[index].amount_of_offset_change();
 
-        return time / 60 % 60;
+        Time res = time / 60 % 60;
+        if (likely(res >= 0))
```

Comment:
How the opposite is possible if we are dividing the number of seconds after midnight (always positive)?

Reply:
I was probably thinking about negative values of `t` (with Datetime64), which will lead to negative `time`  and negative `res`; but that doesn't see to be supported anyway.